src/Umbraco.Core/Configuration/HealthChecks/DisabledHealthCheckElement.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+
+namespace Umbraco.Core.Configuration.HealthChecks
+{
+    public class DisabledHealthCheckElement : ConfigurationElement
```

Comment:
All of these configuration elements should implement an associated interface like we do for the umbraco config so they can be mocked/tested